pvlib/atmosphere.py

```diff
@@ -435,6 +474,6 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
     AMa = airmass_absolute
     modifier = (
         coeff[0] + coeff[1]*AMa  + coeff[2]*pw  + coeff[3]*np.sqrt(AMa) +
-        + coeff[4]*np.sqrt(pw) + coeff[5]*AMa/pw)
+        + coeff[4]*np.sqrt(pw) + coeff[5]*AMa/np.sqrt(pw))
```

Comment:
looks like I made a mistake when I first ported this and put two + signs here (one on each line). Let's keep only the first one.